test/zmumu/tp_from_aod_Data.py

```diff
@@ -235,13 +239,18 @@
 
 
 process.load("MuonAnalysis.TagAndProbe.muon.tag_probe_muon_extraIso_cfi")
+process.load("PhysicsTools.PatAlgos.recoLayer0.pfParticleSelectionForIso_cff")
 
 process.extraProbeVariablesSeq = cms.Sequence(
     process.probeMuonsIsoSequence +
     process.computeCorrectedIso + 
     process.mvaIsoVariablesSeq * process.mvaIsoVariablesTag * process.radialIso +
     process.splitTrackTagger +
-    process.muonDxyPVdzmin 
+    process.muonDxyPVdzmin + 
+    process.muonMiniIsoCharged + 
+    process.muonMiniIsoPUCharged + 
+    process.muonMiniIsoNeutrals + 
+    process.muonMiniIsoPhotons 
```

Comment:
to avoid some copy-paste, maybe you could define a sequence already here, and including also pfParticleSelectionForIsoSequence into it